core/shared/src/main/scala/fs2/Pull.scala

```diff
@@ -43,37 +43,9 @@ final class Pull[+F[_], +O, +R] private (private val free: FreeC[Algebra[Nothing
     */
   def stream(implicit ev: R <:< Unit): Stream[F, O] = {
     val _ = ev
-    Stream.fromFreeC(this.scope.get[F, O, Unit])
+    Stream.fromFreeC(this.asInstanceOf[Pull[F, O, Unit]].get)
```

Comment:
Note converting a `Pull` to a `Stream` no longer inserts a scope -- this is an optional part of this PR but this always felt super arbitrary to me.

Reply:
makes sense